src/components/ReviewComment/ReviewCommentList.tsx

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useState } from 'react';
+import api from '@/Api/api';
+import { Comment } from '@/types';
+import { formatDateTime } from '../../utils/format';
+import { BsTrash } from 'react-icons/bs';
+import { useSetRecoilState } from 'recoil';
+import { createCommentState } from '@/store/store';
+
+type ReviewCommentListProps = {
+  commentList: Comment[];
+};
+
+// pull 하고 지울 것
```

Comment:
```suggestion
// TODO: pull 하고 지울 것 @chunwookJoo 
```
todo tree에 잡히도록하고, userId를 넣어주면 천욱님이 남겼다는걸 알수 있을 것 같네요 :) 

e.g.
![image](https://user-images.githubusercontent.com/30822283/212478011-6d295519-ced2-412f-b977-a2fd54e4f10a.png)

Reply:
와... 이런기능이 있었네요 ㅋㅋㅋ 꿀팁 감사합니다!